lib/couchbase-orm/proxies/bucket_proxy.rb

```diff
@@ -5,11 +5,9 @@
 module CouchbaseOrm
     class BucketProxy
         def initialize(proxyfied)
-            @proxyfied = proxyfied
+            raise "Must proxy a non nil object" if proxyfied.nil?
```

Comment:
don't raise anonymous error. We will create custom couchbase-orm error

Reply:
```suggestion
            raise ArgumentError, "Must proxy a non nil object" if proxyfied.nil?
```